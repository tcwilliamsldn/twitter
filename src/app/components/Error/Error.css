.error {
    display: flex;
    flex-direction: column;
    width: 100vw;
    height: 100vh;
    justify-content: center;
    align-items: center;
    color: #c50000;

    & > h1 {
        font-size: 48px;
        font-weight: bold;
    }

    & > p {
        font-size: 18px;
    }
}
