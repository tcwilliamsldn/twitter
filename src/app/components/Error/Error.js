import React from 'react';
import styles from './Error.css';

const Error = ({ message }) => (
    <div className={styles.error}>
        <h1>Error :(</h1>
        <p>{message}</p>
    </div>
);

export default Error;
