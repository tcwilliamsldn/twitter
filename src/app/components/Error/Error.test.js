import Error from './Error';
import React from 'react';
import { shallow } from 'enzyme';

describe('App', function() {
    it('renders the error message', function() {
        const component = shallow(<Error message="Something's wrong" />);
        expect(component).toMatchSnapshot();
    });
});
