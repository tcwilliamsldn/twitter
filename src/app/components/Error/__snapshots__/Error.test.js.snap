// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders the error message 1`] = `
<div
  className="error"
>
  <h1>
    Error :(
  </h1>
  <p>
    Something's wrong
  </p>
</div>
`;
